mixins/kubernetes/rules/recording_and_alerting_rules/templates/ci_recommended_alerts.json

```diff
@@ -236,6 +236,25 @@
                             }
                         ]
                     },
+                    {
+                        "alert": "Disk space usage for a node is greater than 85%.",
+                        "expression": "1 - avg by (namespace,cluster,job)(node_filesystem_avail_bytes{job=\"node\"}) / avg by (namespace,cluster,job)(node_filesystem_size_bytes{job=\"node\"}) > .85",
+                        "for": "PT5M",
```

Comment:
there is no device & node label level group. So how does the customer know what disk in which node fired this alert ?

Reply:
Thanks Vishwa! I am going to correct this in the other pr.